dspace/config/dspace.cfg

```diff
@@ -1316,7 +1316,9 @@ websvc.opensearch.enable = true
 # url used in service document
 websvc.opensearch.svccontext = opensearch
 # context for html request URLs - change only for non-standard servlet mapping
-websvc.opensearch.uicontext = simple-search
+websvc.opensearch.uicontext = search
+# context for xml request URLs - change only for non-standard servlet mapping
+websvc.opensearch.svccontext = opensearch/search
```

Comment:
This setting appears twice now in `dspace.cfg` with two different values. It's on line 1317 and here.  I suspect lines 1316-1317 should be removed.

Reply:
right. Duplicates have been removed